common/web/types/src/kmx/kmx-plus.ts

```diff
@@ -205,11 +210,15 @@ export class Strs extends Section {
     }
     // nfd
     if (opts?.nfd) {
-      if (opts?.markers) {
-        s = MarkerParser.nfd_markers(s, false);
-      } else {
-        s = s.normalize("NFD");
-      }
+      if (!sections.meta) {
+        throw Error(`Internal Error: need 'meta' section to check normalization mode.`);
```

Comment:
If no `<meta>` section is included, does it default to normalizationDisabled = false?